spec/namespaces/ism.yaml

```diff
@@ -58,6 +110,18 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ism.add_policy@200'
+  /_plugins/_ism/remove:
+    post:
+      operationId: ism.remove.0
+      x-operation-group: ism.remove
```

Comment:
This should be in the same group as `ism.remove_policy`

Reply:
Same as for `change_policy` they have different parameters, one specifies the index in the query, the other in the path.

Reply:
That's correct. But as far as the clients users are concerned, which is what the operation group is for, where the `index` param is put in the URL request is irrelevant. We have many API actions, like search, that also allows the index to be specified as either a path param or a query param, the generated clients will favor use it as a path param regardless.

Reply:
Ok, I had to add `query` to the one that doesn't really take it, but it won't matter from the client POV I think.